README.md

```diff
@@ -21,7 +21,7 @@ npm install grist-api
 const {GristDocAPI} = require('grist-api');
 
 // Put here the URL of your document.
-const DOC_URL = "https://subdomain.getgrist.com/doc/12345678-2761-4ef2-bc28-310e634754fb";
+const DOC_URL = "https://<subdomain>.getgrist.com/doc/12345678-2761-4ef2-bc28-310e634754fb";
```

Comment:
Most document urls don't look like this any more, how about freshening it to something like /123456789abc/My-Document?